docs/source/concepts.rst

```diff
@@ -33,6 +33,24 @@ The entire bootstrapping process may not take longer than (by default) 1800
 seconds before it is considered failed. The timeout can be configured with the
 :envvar:`BOOTSTRAP_TIMEOUT` environment variable.
 
+When creating the CrateDB users specified under ``spec.users``, the Operator
+will add the ``operator.cloud.crate.io/user-password``-label to each of the
+Kubernetes Secrets assigned to one of the users. To keep backward
+compatibility, it also adds the ``operator.cloud.crate.io/user-password``-label
+to each of the existing Kubernetes Secrets used for one of the CrateDB users
+specified in one of the existing CrateDB resources `on resume`_.
+
+The ``operator.cloud.crate.io/user-password``-label is used to filter the
+events when watching for changes on one of the Kubernetes Secrets. If one of
+the Kubernetes Secrets with the ``operator.cloud.crate.io/user-password``-label
+assigned is updated, the Operator will update all CrateDB users that use that
+secret by iterating over all CrateDB resources. The Operator does the update of
+the password in a CrateDB cluster by logging in to one of the master nodes of
+the CrateDB cluster using the username defined in the CrateDB custom resource
+definition and the old password.
+
+Please note that if one updates the CrateDB user directly on one of the master
+nodes, the Operator won't be able to update that user anymore.
```

Comment:
A bit of rewording:

* As far as I can tell, the K8s convention is to start with a `.`.
* "Operator" is not a name and thus not spelled with a capital "O". Unless used in "CrateDB Kubernetes Operator".
* You can use something called [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) to link between Sphinx configurations. The setup for that is in the Sphinx configuration: https://github.com/crate/crate-operator/blob/0cc2d582976b2a21e69674c99b2c958233a07a2d/docs/source/conf.py#L75-L82 . To find the link targets in the target documentation, you can use `python -msphinx.ext.intersphinx URL`, e.g. `python -msphinx.ext.intersphinx https://crate.io/docs/crate/reference/en/stable/objects.inv`. Leveraging intersphinx will help against dead links and adds contextual information to Sphinx what you're linking to. E.g. the "on resume" link looks differently that the one for "ALTER USER", because one is a function (`:func:`) and the other is just a regular reference (`:ref:`). And yeah, I notice, there are other links that should probably use intersphinx.

```suggestion
User Password
~~~~~~~~~~~~~

When creating the CrateDB users specified under ``.spec.users``, the operator
will add the ``operator.cloud.crate.io/user-password`` label to each of the
Kubernetes Secrets assigned to one of the users. To keep backward
compatibility, it also adds the label to Kubernetes Secrets referenced in
existing CrateDB resources :func:`on resume <kopf:kopf.on.resume>`.

The ``operator.cloud.crate.io/user-password`` label is used to filter the
events when watching for changes on one of the Kubernetes Secrets. If one of
the Kubernetes Secrets is updated, the operator will update all CrateDB users
that use that secret by iterating over all CrateDB resources. The operator does
the update of the password in a CrateDB cluster by logging in to one of the
CrateDB nodes with the corresponding username and old password. It will then
use the :ref:`cratedb:ref-alter-user` query to update the password.

.. note::

   If one changes the CrateDB user's password directly in CrateDB, the operator
   won't be able to update that user anymore.
```

Reply:
Thanks for all the explanation!  Intersphinx looks really cool ... never heard about it before.